### PR TITLE
EZP-29618: Update user menu order and add icons

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       |  https://jira.ez.no/browse/EZP-29618
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Related PR: https://github.com/ezsystems/ezplatform-admin-ui/pull/1078

#### Checklist:
- [x] Implement tests
- [x] Coding standards (`$ composer fix-cs`)

### CODE REVIEW

File: src/lib/EventListener/UserMenuListener.php [hunk 1]
```diff
@@ -51,13 +51,16 @@ public function onUserMenuConfigure(ConfigureMenuEvent $event): void
...
+                        'translation_domain' => 'menu', 
+                        'icon' => 'edit',
+                    ],
                 ]
             );
         }
 
         $manipulator = new MenuManipulator();
-        $manipulator->moveToPosition($menu[self::ITEM_CHANGE_PASSWORD], 0);
+        $manipulator->moveToPosition($menu[self::ITEM_CHANGE_PASSWORD], 3);
```
Comment (ViniTou) [inline src/lib/EventListener/UserMenuListener.php]: Not totally sure about this change. Should this option always be on third (even it someone puts something in between) or should it always be after some other specific one?

Maybe this whole menu should use same approach as in flex and date-based publisher - custom `orderNumber` attribute to sort those?

@konradoboza

Comment (konradoboza) [reply src/lib/EventListener/UserMenuListener.php]: That's correct, we put each menu item every 10 positions to allow to customize this by placing custom items in between. Consistent-wise I think this should be also the case in the user menu.

Comment (tischsoic) [reply src/lib/EventListener/UserMenuListener.php]: I changed approach in commit: https://github.com/ezsystems/ezplatform-user/pull/46/commits/6cf779b907533961c7e2df4ee63c9ee4e56be3c8
Can you check it, @konradoboza @ViniTou?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 /vendor
 /phpunit.xml
 /.php_cs.cache
+composer.lock
```
Comment (ViniTou) [inline .gitignore]: :)

File: src/lib/EventListener/UserMenuListener.php (hunk 1 above)
Comment (konradoboza) [reply src/lib/EventListener/UserMenuListener.php]: Why `35`? :) I think it should be `40`.  To be consistent: `50` for `self::ITEM_USER_SETTINGS` and `60` for`self::ITEM_LOGOUT` in the corresponding PR.

Comment (tischsoic) [reply src/lib/EventListener/UserMenuListener.php]: I wanted _ezplatform-admin-ui_ to behave as if _ezplatform-user_ didn't exist. I will change this. :)

**Comment by tischsoic** [comment]
I have to thank @konradoboza for detailed explanation on how ordering of menu items should be implemented. :)

**Pull request merged**